packages/core/helper-plugin/src/components/CheckPagePermissions.tsx

```diff
@@ -15,7 +15,10 @@ export interface CheckPagePermissions {
   permissions?: Permission[];
 }
 
-const CheckPagePermissions = ({ permissions = [], children }: CheckPagePermissions) => {
+const CheckPagePermissions: React.FC<CheckPagePermissions> = ({
```

Comment:
```suggestion
const CheckPagePermissions = ({
```

I think we decided a while back we weren't doing FC 🤔 how come you feel you need it?